modules/ldap_groups/app/views/ldap_groups/synchronized_groups/show.html.erb

```diff
@@ -10,6 +10,35 @@
        nested_breadcrumb_element(t("ldap_groups.synchronized_groups.singular"), @group.dn)],
       selected_item_font_weight: :normal
     )
+
+    unless @group.filter
```

Comment:
Hi @oliverguenther 
I just have a question, for showing edit button why did you check filter?

Reply:
because in that case, the group is not editable, as it's provided/overridden by the filter